static/src/javascripts/projects/membership/contributions-recurring-tab.js

```diff
@@ -40,6 +42,43 @@ const displayErrorMessage = (): void => {
     $(ERROR).removeClass(IS_HIDDEN_CLASSNAME);
 };
 
+const displayCancelContribution = (): void => {
+    $(CANCEL_CONTRIBUTION).removeClass(IS_HIDDEN_CLASSNAME);
+};
+
+const hideCancelContribution = (): void => {
+    $(CANCEL_CONTRIBUTION).addClass(IS_HIDDEN_CLASSNAME);
```

Comment:
Double checking that this works when the switch is off because the lack of element gives you a bonzo wrapped set of 0 elements instead of an undefined/null?

Reply:
I checked in CODE and this function is not being called with the switch off.